python/paddle/utils/merge_model.py

```diff
@@ -70,4 +70,4 @@ def merge_v2_model(net, param_file, output_file):
         for pname in param_names:
             params.serialize(pname, f)
 
-    print 'Generate  %s  success!' % (output_file)
+    print('Generate  %s  success!' % (output_file))
```

Comment:
from __future__ import print_function

Reply:
This is the usage of python2, which is not encouraged in python3, change the print is the right way